### PR TITLE
Door panels

### PR DESCRIPTION
**Pull request by jotpowers**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

This creates files for doors on 2.4 (but would probably work on others)

#### How was it tested? 

Installed on my printer.

#### Any background context you want to provide?

Hopefully all in the README.

#### Screenshots (if appropriate)

Photos included in the PR.

### CODE REVIEW

**Comment by jotpowers** [comment]
Ok, hopefully this is cleaner in its own branch.

**Review by clee — APPROVED** [review]
STL appears to be in the correct orientation, commit history is no longer a tangled mess, no glaring typos or grammatical errors in the README on first skim, files included all appear to be relevant… I can't check the CAD files on this machine but everything else looks good to me.

File: printer_mods/gamebofh/door-panels/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Laser Cut Door Panels
+
+As my distaste for all solutions VHB related, I was looking for how to replace the hinges on my 2.4.  The most popular solution seems to be the [270 Degree Hinge](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/chrisrgonzales/270_degree_hinge) which was based on [Randell's Door Hinges](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/randell/Door_Hinges).
```
Comment (cbc02009) [inline printer_mods/gamebofh/door-panels/README.md:3]: please change these to relative links (../../chrisgonzales/270_degree_hinge for example)

**Review by cbc02009 — CHANGES_REQUESTED** [review]
are you missing STLs for the actual hinges, or are they meant to be used from another mod?

File: printer_mods/gamebofh/door-panels/README.md (hunk 1 above)
Comment (jotpowers) [reply printer_mods/gamebofh/door-panels/README.md]: Made the changes to those two links.  Don't know what I need to do to the PR though.

In response to your earlier question, yes the hinges are a different mod, but it's these links so hopefully that answers it.

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**